website/docs/source/v2/provisioning/ansible_intro.html.md

```diff
@@ -192,14 +208,17 @@ variable2=example
   - Prior to Vagrant 1.7.3, the `ansible_ssh_private_key_file` variable was not set in generated inventory, but passed as command line argument to `ansible-playbook` command.
   - The generation of group variables blocks (e.g. `[group1:vars]`) is only possible since Vagrant 1.8.0. Note however that setting variables directly in the inventory is not the [preferred practice in Ansible](http://docs.ansible.com/intro_inventory.html#splitting-out-host-and-group-specific-data). If possible, group (or host) variables should be set in `YAML` files stored in the `group_vars/` or `host_vars/` directories in the playbook (or inventory) directory instead.
   - Unmanaged machines and undefined groups are not added to the inventory, to avoid useless Ansible errors (e.g. *unreachable host* or *undefined child group*)
+  - [Host range patterns (numeric and alphabetic ranges)](http://docs.ansible.com/ansible/intro_inventory.html#hosts-and-groups) will not be validated by vagrant. Hosts will be added as group members to the inventory anyway, this might lead to errors in Ansible (e.q *unreachable host*).
 
-For example, `machine3` and `group3` in the example below would not be added to the generated inventory file:
+For example, `machine3`, `group3` and `group1:vars` in the example below would not be added to the generated inventory file,
```

Comment:
This might be a rebase issue after #6627, please look at https://github.com/mitchellh/vagrant/pull/6627/files#diff-2bd299aaa9055065bd144dc08f176a8bR196 and fix it accordingly.